test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Can you write 2 tests here:
1. Check the response status (:ok)
2. Check that the content returned is correct ("Hello World")

Reply:
done;)